features/aragon/aragonForm.tsx

```diff
@@ -0,0 +1,111 @@
+import { Block, Button, ToastError } from '@lidofinance/lido-ui';
+import { useAragonVote } from 'features/vesting';
+import { SelectVesting } from 'features/vesting/selectVesting';
+import { useCallback } from 'react';
+import { useForm } from 'react-hook-form';
+import { InputGroupStyled, InputNumber } from 'shared/ui';
+import { useEncodeAragonCalldata } from 'features/votingAdapter';
+import { Form } from './aragonFormStyles';
+import { stringToNumber } from 'shared/lib';
+import { useGetVoting } from './useAragon';
+
+type AragonFormData = {
+  voteId: string;
+  success: boolean;
+};
+
+const validateVoteId = (value: string) => {
+  const number = Number(value);
```

Comment:
```suggestion
  const number = parseInt(value);
```

as you do not want to accidentally get floats

Reply:
Thanks, going to fix 👍